var/spack/repos/builtin/packages/suite-sparse/package.py

```diff
@@ -75,47 +83,28 @@ def install(self, spec, prefix):
             # completely disabled. See
             # [SuiteSparse/SuiteSparse_config/SuiteSparse_config.mk] for more.
             'CUDA=no',
-            'CUDA_PATH={0}'.format(
-                spec['cuda'].prefix if '+cuda' in spec else ''
-            ),
-            'CFOPENMP={0}'.format(self.compiler.openmp_flag)
-        ])
-
-        if '+pic' in spec:
-            make_args.extend([
-                'CFLAGS={0}'.format(self.compiler.pic_flag),
-                'FFLAGS={0}'.format(self.compiler.pic_flag)
-            ])
+            'CUDA_PATH=%s' % (spec['cuda'].prefix if '+cuda' in spec else ''),
+            'CFOPENMP=%s' % omp_flag,
+            'CFLAGS=-O3 -fexceptions -fno-common %s' % pic_flag,
```

Comment:
Unfortunately it does not necessarily mean that it works in mentioned cases. Unless u can confirm, I would not change it as it would be a shame to brake Suitesparse for other users